### PR TITLE
Migrate domain name from freecadweb to freecad

### LINKED ISSUE #6415: [Task] search and replace URLs to freecad.org
### Issue description

Once v0.21dev is cut, migrate all freecad URLs to use the freecad.org domain name.

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by luzpaz**
Pre-v0.21 release change to FreeCAD master branch migrating away from `freecadweb.org` to `freecad.org`  domain. Migration broken down in to separate commits (per request) for easier review.  Note there are some remaining spec files that haven't been changed since they will break spec.

Closes #6415

---

- [X]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

### CODE REVIEW

File: package/fedora/freecad.spec [hunk 1]
```diff
@@ -39,7 +39,7 @@ Summary:        A general purpose 3D CAD modeler
 Group:          Applications/Engineering
 
 License:        LGPLv2+
-URL:            http://www.freecadweb.org/
+URL:            http://www.freecad.org/
```
Comment (luzpaz) [inline package/fedora/freecad.spec:42]: This change seems safe and therefore was included

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9352">pipeline status</a> for feature branch [PR_9352](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9352). Pipeline [845346407](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/845346407) was triggered at [43cfb82](https://github.com/FreeCAD/FreeCAD/pull/9352/commits/43cfb8255414963424a9ae163aa9e625eca5f0bc). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by luzpaz** [comment]
Rebased.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9352">pipeline status</a> for feature branch [PR_9352](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9352). Pipeline [845686748](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/845686748) was triggered at [114f4c9](https://github.com/FreeCAD/FreeCAD/pull/9352/commits/114f4c9388b45d7bf664c5601865b83490bb5dc7). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9352">pipeline status</a> for feature branch [PR_9352](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9352). Pipeline [845689358](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/845689358) was triggered at [b5f76a4](https://github.com/FreeCAD/FreeCAD/pull/9352/commits/b5f76a4ad65b2e3b8f5020e7ecd95bdd90029620). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by luzpaz** [comment]
Merge dev, please squash commits after review and merge so can add commit hash to the git blame ignore file. TIA!

**Comment by chennes** [comment]
I think that rather than squashing, let's do a merge commit. So you can add the hashes as part of this PR if you want, they won't change.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9352">pipeline status</a> for feature branch [PR_9352](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9352). Pipeline [845927295](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/845927295) was triggered at [ddae2aa](https://github.com/FreeCAD/FreeCAD/pull/9352/commits/ddae2aa449ecb2da3059df6e1112a93884790617). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by chennes** [comment]
@luzpaz The reformat of Tools gives a couple of trivial conflicts -- can you resolve?

**Comment by luzpaz** [comment]
Thanks :)

I used the github conflict resolver because I was getting conflicting results with the web editor but now am trying to squash the last commit (unsuccessfully so far).

**Comment by chennes** [comment]
Let's not worry about it too much - this is a big set of changes, I don't want it floating out there for too long or you'll just get more conflicts. OK if I merge?

**Comment by luzpaz** [comment]
Just squash the last commit on to the previous one?

**Comment by chennes** [comment]
In the GitHub interface squash is an all-or-nothing operation -- I'll just ahead and do that.

**Pull request merged**